src/RCAT_main.py

```diff
@@ -43,13 +42,23 @@ def get_settings(config_file):
         'map kwargs': conf_dict['PLOTTING']['map kwargs'],
         'line grid setup': conf_dict['PLOTTING']['line grid setup'],
         'line kwargs': conf_dict['PLOTTING']['line kwargs'],
-        'nodes': conf_dict['SLURM']['nodes'],
-        'slurm kwargs': conf_dict['SLURM']['slurm kwargs'],
+        'cluster type': conf_dict['CLUSTER']['cluster type'],
+        'nodes': conf_dict['CLUSTER']['nodes'],
+        'cluster kwargs': conf_dict['CLUSTER']['cluster kwargs'],
         'outdir': conf_dict['SETTINGS']['output dir'],
     }
 
     return d
 
+def local_cluster_setup():
+    """
+    Set up local-pc cluster
+
+    """
+    from dask.distributed import LocalCluster
+    cluster = LocalCluster(processes=False)
```

Comment:
Why is there no **kwargs option for LocalCluster? Should processes always be set to False?

Reply:
It's because I couldn't get it working without processes=False. Without it, the different threads start from the top of RCAT_main.py and then it crashes. It would be good if you can test it as well.

Reply:
I've tested running locally. It does work fine with processes=False but not with True. We should initially force this option to be False.